installing/installing_restricted_networks/installing-restricted-networks-bare-metal.adoc

```diff
@@ -92,6 +92,5 @@ include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
 
 .Next steps
 
+* xref:../../telemetry/opting-out-of-telemetry.adoc#opting-out-of-telemetry[Opt out of telemetry].
```

Comment:
This sounds like "and then you'll want to...".  Is there some phrasing that emphasizes the optional-ness?  Maybe "Optional next steps" for the section title?